docs/setup-steps-1.4.0.md

```diff
@@ -0,0 +1,265 @@
+
+       ![](\_images/ctk-getCertificate.png)
+       
+   *   This certificate is to be used by **SBI** devices as the encryption key.
+
+       For Mock **MDS**, when running in **Auth** mode: update the below values in the `application.properties` file.
+
+       ```
+       mosip.auth.appid=regproc
+       mosip.auth.clientid=mosip-pms-client
+       mosip.auth.secretkey=XXXXXXXXXXXXXXXX
+       mosip.auth.server.url=https://api-internal.${env}.mosip.net/v1/authmanager/authenticate/clientidsecretkey 
+       mosip.ida.server.url=https://api-internal.${env}.mosip.net/v1/keymanager/getCertificate?applicationId=COMPLIANCE_TOOLKIT&referenceId=COMP-FIR
+       ```
+
+       For real MDS/SBI, the vendors can download the new encryption key from the UI and test with the updated **SBI** which uses this encryption key. It can be downloaded for **Auth SBI** projects from UI.
+
+       ![](\_images/ctk-encryptionkey.png)
+       
+6. Ensure that `reporting` module is deployed from the `develop` branch. This is required for the **Kibana Dashboard**.
+
+7. Ensure that the `kernel-auth-adapter-1.2.0.1` has been successfully deployed in the `compliance-toolkit-service`, and verify that the identical authentication adapter was utilized to configure the mock ABIS.
+
+8. Check datashare configurations for **ABIS3030** and **ABIS3031** testcases.
+
+    * Ensure that in `data-share-default.properties` and `compliance-toolkit-default.properties`, the value of `mosip-abis-client` is set in `auth.server.admin.allowed.audience`. If this was not set by default, then set it and restart `data-share-service` and `compliance-toolkit-service`.
+
+    * For handle CTK datashare token flow, add the below values in `data-share-default.properties`.
+
+        ```
+        auth.handle.ctk.flow=true
+        mosip.api.internal.toolkit.url=https://${mosip.api.internal.host}/v1/toolkit
+        mosip.compliance.toolkit.saveDataShareToken.url=${mosip.api.internal.toolkit.url}/saveDataShareToken
+        mosip.compliance.toolkit.invalidateDataShareToken.url=${mosip.api.internal.toolkit.url}/invalidateDataShareToken
+        mosip.compliance.toolkit.invalidateDataShareToken.testCaseId=ABIS3031
+        ```
+
+
+### Steps to load testdata, schemas in MINIO
+
+1\. Browse [mosip-compliance-toolkit](https://github.com/mosip-compliance-toolkit.git)
+
+2\. Project structure will be as shown below.
+
+![](\_images/ctk-project-structure.png)
+
+3\. The resources folder has schemas and test data that need to be added to MinIO.
+
+![](\_images/ctk-resources-folder-new.png)
+
+#### Steps to load testdata and schemas directly in MinIO
+
+1\. Log in to MinIO from the browser.
+
+2\. Create a `compliance-toolkit` bucket.
+
+3\. Create a new folder named `testdata` in the above bucket.Upload MOSIP\_DEFAULT\_XXX.zip files from resources to it.
```

Comment:
not clear. Please re-write sentence to clearly bring out what 'resources' are referred to?